### DISCUSSION TITLE
Discord cannot find osu lazer application when fullscreen

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#18631] [Q&A] [answered]
Works without fullscreen, but adds latency, annoying vicious circle.
![image](https://user-images.githubusercontent.com/40531871/172966608-ce691093-db16-485c-ba4b-09af254f43ac.png)
Note : Sharing through this option works, but screen is frozen, only music plays.
![image](https://user-images.githubusercontent.com/40531871/172966744-12647924-5848-4bbe-89c6-1312ca865652.png)

**Comment by peppy** [comment]
Works fine here, FWIW.

**Reply by AuracleTech** [reply]
I installed discord canary to see if it was my discord installation, it's not. same thing on discord canary, my friends see the game frozen.
![image](https://user-images.githubusercontent.com/40531871/172979803-9d8f64c2-b225-4282-b90a-f8e5ae0955e6.png)
Their POV
![image](https://user-images.githubusercontent.com/40531871/172979877-beadcefb-2c5d-489d-8d7a-13e245321d3c.png)

**Comment by smoogipoo** [comment]
The game not showing is just how discord works when applications are minimised. Still unsure whether we want to be minimising the app when alt-tabbing from fullscreen, but it is what it is.

The game being frozen should be reported to discord instead.

**Reply by AuracleTech** [reply]
Considering every other application works but not osu lazer I don't even know if they will take the request seriously.

**Reply by smoogipoo** [reply]
Put another way, I don't think we can fix this from our side.

**Comment by Theighlin** [comment]
I had the same problem and updatingt he graphic card drivers solved it for me (dual gpu laptop, solved it whether the game is running on integrated or dedicated gpu)

**Reply by AuracleTech** [reply]
Opened up the game, streamed it, worked.
Alt tabbed, stopped working.
Restarted the stream, stopped working right away.

It appears if I ALT TAB even once, the stream will never work again up until I restart the whole game. (Osu fullscreen only)
Note : I'm on a desktop GTX 1060, drivers are up to date, tested with 2 discord versions (Official and Canary)

**Comment by AuracleTech** [comment, marked as answer]
I fixed the issue by upgrading to Windows 11
I was on Windows 10 when issue happened
